packages/electrode-archetype-opt-mocha/optional-check.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+const Path = require("path");
+const assert = require("assert");
+
+const cwd = process.env.PWD || process.cwd();
+
+function lookupAppDirByInitCwd() {
```

Comment:
@scrajesh can we pls start DRY-ing this logic? It seems to be present in multiple places

Reply:
can do but in a diff story and sprint.